### PR TITLE
[5.6] Add more uncountable nouns

### PR DESCRIPTION
**Pull request by TheAlexLichter**

### CODE REVIEW

**Comment by rs-sliske** [comment]
unless im getting confused with that does, im not sure all of those are correct

- businesses
- damages
- failures
- successes
- times
- usages

**Comment by devcircus** [comment]
This isn't right

**Comment by manniL** [comment]
Oops. Corrected!

**Comment by devcircus** [comment]
Accommodations

**Comment by manniL** [comment]
@devcircus I thought accommodation has no plural form, but it looks like it's used sometimes.

File: src/Illuminate/Support/Pluralizer.php [hunk 1]
```diff
@@ -12,28 +12,39 @@ class Pluralizer
      * @var array
      */
     public static $uncountable = [
+        'access',
         'audio',
         'bison',
+        'carbon',
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Pluralizer.php]: This is countable.

File: src/Illuminate/Support/Pluralizer.php [hunk 2]
```diff
@@ -12,28 +12,39 @@ class Pluralizer
      * @var array
      */
     public static $uncountable = [
+        'access',
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Pluralizer.php]: So is this.

File: src/Illuminate/Support/Pluralizer.php [hunk 3]
```diff
@@ -12,28 +12,39 @@ class Pluralizer
...
         'audio',
         'bison',
+        'carbon',
+        'cash',
         'chassis',
+        'chess',
+        'commerce',
         'compensation',
+        'content',
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Pluralizer.php]: So is this.

File: src/Illuminate/Support/Pluralizer.php [hunk 4]
```diff
@@ -12,28 +12,39 @@ class Pluralizer
...
+        'chess',
+        'commerce',
         'compensation',
+        'content',        
         'coreopsis',
         'data',
         'deer',
         'education',
         'emoji',
+        'energy',
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Pluralizer.php]: So is this.

File: src/Illuminate/Support/Pluralizer.php [hunk 5]
```diff
@@ -12,28 +12,39 @@ class Pluralizer
...
         'firmware',
         'fish',
         'furniture',
+        'garbage',
         'gold',
         'hardware',
         'information',
         'jedi',
         'knowledge',
+        'logic',
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Pluralizer.php]: So is this. Definitely!

**Pull request closed without merging**